src/pages/Leaderboard/Leaderboard.test.tsx

```diff
@@ -99,7 +99,7 @@ test("render the search bar", () => {
 });
 
 function mockMember() {
-  const userName = faker.internet.userName();
+  const userName = faker.internet.username();
```

Comment:
테스트를 실행했더니 다음과 같은 depreaction waring이 뜨고 있어서 조치하였습니다.

```sh
[@faker-js/faker]: faker.internet.userName() is deprecated since v9.1.0 and will be removed in v10.0.0. Please use faker.internet.username() instead.
```